engine/time/src/main/java/io/deephaven/time/DateTime.java

```diff
@@ -49,6 +49,8 @@ public static DateTime of(Instant instant) {
         return new DateTime(DateTimeUtils.nanos(instant));
     }
 
+    public static final DateTime[] EMPTY_DATETIME_ARRAY = new DateTime[0];
```

Comment:
We had one in `DateTimeUtils` already (`io.deephaven.time.DateTimeUtils#ZERO_LENGTH_DATETIME_ARRAY`).